pages/order-list/index.js

```diff
@@ -1,17 +1,37 @@
 import ShoppingCart from "/components/shopping-cart";
 import Drinks from "/components/order-menu/drink-menu";
 import Maindishes from "/components/order-menu/main-menu";
+import OrderOverview from "../order-overview";
+import Backdrop from "@/components/back-drop/index.js";
 import BackButton from "@/components/back-button";
+import { useState } from "react";
 
 export default function OrderList({ handleAmountChange, items }) {
+  const [drawerOpen, setDrawerOpen] = useState(false);
+
+  const drawerToggleClickHandler = () => {
+    setDrawerOpen(!drawerOpen);
+  };
+
+  const backdropClickHandler = () => {
+    setDrawerOpen(false);
+  };
+
+  let backdrop;
+  if (drawerOpen) {
+    backdrop = <Backdrop close={backdropClickHandler} />;
+  }
```

Comment:
This can be done more succinctly. Remove these lines entirely and then see the next comment.